README.md

```diff
@@ -11,4 +11,107 @@ planed are:
 
 Please check the API documentation for more details
 
-**PRs  for other collections or test are very welcome**
\ No newline at end of file
+**PRs  for other collections or test are very welcome**
+
+#### Example for ListNotifier:
+
+
+```Dart
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:listenable_collections/listenable_collections.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+      ),
+      home: MyHomePage(title: 'List Notifier Example'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  var _list = ListNotifier();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: SingleChildScrollView(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              actionsButtonMenu,
+              ValueListenableBuilder(
+                valueListenable: _list,
+                builder: (context, value, child) => Column(
+                  mainAxisSize: MainAxisSize.max,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: List.from(_list.map((value) => Text('List Item: $value'))),
```

Comment:
Why did you use List.from and not just add a ´toList()` at the end?

Reply:
I used to List.from to generate list of widgets (in this case list of Text widget). Couldn't use `_list.toList()` because `_list` its type of `<String>`

Reply:
`_list.map<Widget>((value) => Text('List Item: $value').toList()`
should work because it's after the Map

Reply:
Yep its working, check my latest commit.